lib/simulator-xcode-6.js

```diff
@@ -457,6 +462,12 @@ class SimulatorXcode6 extends EventEmitter {
       args.push('-ConnectHardwareKeyboard', opts.connectHardwareKeyboard ? '1' : '0');
     }
 
+    if (_.isBoolean(opts.disablePasteboardAutomaticSync)) {
```

Comment:
this will not work if opts.disablePasteboardAutomaticSync is set to `false`: the `-PasteboardAutomaticSync` is always set to `0`

Reply:
Oops... true. it's my mistake...